### ISSUE TITLE
Keyboard focus indicator lost in left rail menu; menu headings not expandable in istio.io/docs

### ISSUE THREAD

**Issue by Lealikeintea** [istio/istio.io#7426]
When navigating through istio.io/docs via keyboard using the tab key, the focus indicator is not visible in the left side rail menu and the side menu headings are not able to be expanded, violating three WCAG accessibility principles (linked below).

[2.4.3 - Focus Order (Level A)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#focus-order) - If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability.

[2.4.7 - Focus Visible (Level AA)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#focus-visible) - Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible.

[2.4.8 - Location (Level AAA)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#location) - Information about the user's location within a set of Web pages is available.
